pkg/controller/volume/persistentvolume/deprecated-recycle-api.md

```diff
@@ -0,0 +1,32 @@
+## Deprecation of Persistent Volume Recycler Behavior
```

Comment:
@kubernetes/docs I know the release is imminent, but I want to put this into the queue.  Do we have a doc or index of docs that cover deprecated features?